src/billing/Members/Members.tsx

```diff
@@ -1,57 +1,46 @@
 import _ from 'lodash/fp';
 import React, { ReactNode, useState } from 'react';
 import { billingRoles } from 'src/billing/utils';
-import { DeleteUserModal } from 'src/components/DeleteUserModal';
-import { EditUserModal } from 'src/components/EditUserModal';
-import { MemberCard, MemberCardHeaders, NewUserCard, Sort, User } from 'src/components/group-common';
-import { NewUserModal } from 'src/components/NewUserModal';
+import { DeleteMemberModal } from 'src/groups/Members/DeleteMemberModal';
+import { EditMemberModal } from 'src/groups/Members/EditMemberModal';
+import { Member, MemberTable } from 'src/groups/Members/MemberTable';
+import { NewMemberModal } from 'src/groups/Members/NewMemberModal';
 import { Ajax } from 'src/libs/ajax';
 import { BillingRole } from 'src/libs/ajax/Billing';
 
 interface MembersProps {
   billingProjectName: string;
   isOwner: boolean;
-  projectUsers: User[];
+  projectUsers: Member[];
```

Comment:
Do you want to go all the way and rename things like `projectUsers` to `projectMembers`?

Reply:
good idea